bin/propolis-server/src/lib/stats/pvpanic.rs

```diff
@@ -4,13 +4,14 @@
 
 use super::virtual_machine::VirtualMachine;
 use chrono::Utc;
-use oximeter::{
-    types::{Cumulative, Sample},
-    Metric, MetricsError, Producer,
-};
+use oximeter::{types::Sample, Metric, MetricsError, Producer};
 use propolis::hw::qemu::pvpanic;
 use std::sync::Arc;
 
+oximeter::use_timeseries!("virtual-machine.toml");
```

Comment:
Should we be expecting a `virtual-machine.toml` be included in the repo as part of this change?

Reply:
No, that lives [here](https://github.com/oxidecomputer/omicron/blob/main/oximeter/oximeter/schema/virtual-machine.toml) in Omicron. Part of the work to support updates to timeseries schema includes centralizing them in Omicron. There are obviously trade-offs here, and this is where we've landed for now.

Reply:
I think it might be worth including a comment on this line explaining that, so that someone reading this file knows where to look to see the schema?

Reply:
Commented in 4d5453a